analysis/import/SCMLogParserV2/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparserv2/input/Commit.kt

```diff
@@ -0,0 +1,26 @@
+package de.maibornwolff.codecharta.importer.scmlogparserv2.input
+
+import java.time.OffsetDateTime
+
+class Commit(
+    val author: String,
+    modifications: List<Modification>,
+    val commitDate: OffsetDateTime,
+    private val mergeCommit: Boolean = false
+) {
+
+    val modifications: List<Modification>
+
+    val filenames: List<String>
```

Comment:
Refactor suggestion: 
new property fileNamesList
if fileNamesList empty
  iterate modifications and map
else 
  return fileNamesList